### ISSUE TITLE
Unreadable docker logs

### ISSUE THREAD

**Issue by NehzUx** [codalab/codabench#642]
I created a gpu worker and successfully runned gpu submissions on v2. However, when I check the docker logs, it is unreadable as below. Do you why there are so many `\x08` in the logs? Thanks for having a check @ckcollab 

> x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\r128/234 [===============>..............] - ETA: 0s - loss: 0.1098 - acc: 0.9766\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\r224/234 [===========================>..] - ETA: 0s - loss: 0.0972 - acc: 0.9821\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\r234/234 [==============================] - 0s 805us/sample - loss: 0.0938 - acc: 0.9829 - val_loss: 3.2355 - val_acc: 0.5091\nEnd   [train]. Time elapsed: 0.77 sec.\n2021-01-21 11:27:00,471 INFO ingestion.py: [01-21 11:27:00] training success, time spent so far 763.5114657878876 sec\n2021-01-21 11:27:00,472 INFO ingestion.py: Finished training the model.\n2021-01-21 11:27:00,472 INFO ingestion.py: Begin testing the model by making predictions on test set...\nStart [test]:\n----cur_max_auc 0.6812895257393738; cur_auc 0.6144501412858288; lstm auc not rise for 1 times\n----not predict\n----model_num: 37 Select k best 5 predicts which have auc [0.6906533951642122, 0.6888370705256097, 0.688031778195912, 0.6823878203763776, 0.6812895257393738], each model 3 best which have auc [0.6906533951642122, 0.6812895257393738, 0.6780558095578657], and each previous model\'s best predict which have auc [\'(lr:[0.5015878975703756])\', \'(lstm:[0.6780558095578657, 0.4854235339525883, 0.6040591376030318, 0.6198877659810971, 0.6386898551382729, 0.5860587724451483, 0.6402508899540904, 0.632202914262632, 0.6381093487040206, 0.6667436061895209, 0.5966659713530804, 0.6812895257393738])\', \'(crnn:[0.46826902770564494, 0.4318129748353243, 0.45632702915681644, 0.4445453614069743, 0.4642944184766107, 0.48811217915062, 0.5463783943837583, 0.4967736825226545, 0.4477582730491725, 0.49504946754115364, 0.4905240517010587])\', \'(bilstm:[0.6906533951642122, 0.5012112476196153, 0.6539644150804529, 0.6715954665554165, 0.6620571519387897, 0.6492621786303172, 0.6396363538323131, 0.6452937780313068, 0.6089037279214287, 0.5690200865216062, 0.5570754571332079])\'] \nEnd   [test]. Time elapsed: 0.02 sec.\n2021-01-21 11:27:00,494 INFO ingestion.py: [01-21 11:27:00] predicting success, time spent so far 763.5334317684174 sec\n2021-01-21 11:27:00,494 INFO ingestion.py: Finished making predictions.\n2021-01-21 11:27:00,496 INFO ingestion.py: [+] 403 predictions made, time spent so far 768.06 sec\n2021-01-21 11:27:00,496 INFO ingestion.py: [+] Time left 1036.47 sec\n2021-01-21 11:27:00,496 INFO ingestion.py: Begin training the model...\nStart [train]:\n----fit lstm for 11 times\n----Start [get_train_data]:\n--------Start [extract_mfcc_parallel]:\n--------End   [extract_mfcc_parallel]. Time elapsed: 0.00 sec.\n--------Total 234, update 0\n----End   [get_train_data]. Time elapsed: 0.01 sec.\n----train_x: (234, 227, 128); train_y: (234, 9); val_x: (55, 227, 128); val_y: (55, 9);\n----This train loop use lstm, last train loop use lstm\nTrain on 234 samples, validate on 55 samples\nEpoch 1/10\n\r 32/234 [===>..........................] - ETA: 0s - loss: 0.0602 - acc: 0.9688\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\r128/234 [===============>..............] - ETA: 0s - loss: 0.0919 - acc: 0.9609\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\r224/234 [===========================>..] - ETA: 0s - loss: 0.1532 - acc: 0.9509\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\r234/234 [==============================] - 0s 777us/sample - loss: 0.1468 - acc: 0.9530 - val_loss: 3.3838 - val_acc: 0.4727\nEpoch 2/10\n\r 32/234 [===>..........................] - ETA: 0s - loss: 0.0316 - acc: 1.0000\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\r128/234 [===============>..............] - ETA: 0s - loss: 0.0841 - acc: 0.9844\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\r224/234 [===========================>..] - ETA: 0s - loss: 0.0605 - acc: 0.9866\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\r234/234 [==============================] - 0s 777us/sample - loss: 0.1136 - acc: 0.9786 - val_loss: 3.7192 - val_acc: 0.4545\nEpoch 3/10\n\r 32/234 [===>..........................] - ETA: 0s - loss: 0.1701 - acc: 0.9688\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08

**Comment by ckcollab** [comment]
mind sending the competition bundle + a submission to the #codabench channel on our shared slack?

**Comment by Logan-Ruf** [comment]
Testing this locally, I'm not getting the same issue. Could be something to do with the competition or submission outputting the wrong string format?

**Comment by NehzUx** [comment]
I find that this only occurs when ssh to VM using web browser. If I use my own terminal, it shows normally. I close this issue for the moment.

**Comment by NehzUx** [comment]
Sorry for the wrong information. After I checked more on my own terminal, this problem also occurs. I am using this challenge https://www.codabench.org/competitions/238/#/participate-tab. It is very large (several G), you can access the bundle here 
![image](https://user-images.githubusercontent.com/12370705/105814481-71236e00-5fec-11eb-8174-1f9a8542b841.png)


> [2021-01-26 07:33:43,103: WARNING/ForkPoolWorker-1] WS: b'\r  32/1120 [..............................] - ETA: 0s - loss: 22.6313 - acc: 0.0000e+00\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\r 128/1120 [==>...........................] - ETA: 0s - loss: 22.9357 - acc: 0.0000e+00\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\r 224/1120 [=====>........................] - ETA: 0s - loss: 23.6371 - acc: 0.0045    \x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\r 320/1120 [=======>......................] - ETA: 0s - loss: 23.9190 - acc: 0.0094\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\r 416/1120 [==========>...................] - ETA: 0s - loss: 23.6402 - acc: 0.0096\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\r 512/1120 [============>.................] - ETA: 0s - loss: 23.5538 - acc: 0.0078\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\r 608/1120 [===============>..............] - ETA: 0s - loss: 23.4143 - acc: 0.0115\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\r 704/1120 [=================>............] - ETA: 0s - loss: 23.3369 - acc: 0.0128\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\r 800/1120 [====================>.........] - ETA: 0s - loss: 23.3572 - acc: 0.0137\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\r 896/1120 [=======================>......] - ETA: 0s - loss: 23.2514 - acc: 0.0134\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\r 992/1120 [=========================>....] - ETA: 0s - loss: 23.4693 - acc: 0.0121\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\r1088/1120 [============================>.] - ETA: 0s - loss: 23.3904 - acc: 0.0110\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\r1120/1120 [==============================] - 1s 714us/sample - loss: 23.3607 - acc: 0.0107 - val_loss: 12.3147 - val_acc: 0.0188\n'
> [2021-01-26 07:33:43,205: WARNING/ForkPoolWorker-1] WS: b'Epoch 4/10\n'
